installers/conf/config_sync.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -z ${PGOROOT} ]
+then
+	echo "error: \$PGOROOT must be set"
+	exit 1
+fi
+
+INSTALLER_DIR="$PGOROOT/installers"
+CONF_DIR="$INSTALLER_DIR/conf"
+TEMPLATE_DIR="$CONF_DIR/templates"
+
+if [[ -d ${INSTALLER_DIR} || -d ${CONF_DIR} || -d ${TEMPLATE_DIR} ]]
```

Comment:
Is this sort of check necessary? This would be a developer script but could also be used with CI. 

I'm thinking that you'd have to be doing something really weird to have those files not exist in `$PGOROOT`

Reply:
Since these directories are part of the actual repo, I think we can safely assume they'll always be there (i.e. no user action is really needed to be able to run this script, and they would need to explicitly delete the referenced dirs to break it).

Reply:
Thats what I was thinking, alright I'm gonna remove the checks then.